### ISSUE TITLE
Game analysis bug?

### ISSUE THREAD

**Issue by bornwicked** [EdNekebno/DGTCentaurMods#143]
I have been experiencing sporadic strange crashes. Typically I see this when playing engine ct800: The screen flickers between 2 setups, one where the pieces are unmoved and one where they correctly shows the current game. This is annoying but I am used to it. However, today the engine moved twice, without waiting for my move! I noticed that the "unmoved" chessboard setup actually registered my move, but the other didn't. It seems that I have 2 engines running simultaneously on my board... and this is very likely what is causing the sporadic crashes for me. My theory is that one engine typically just lies in the background, but is sometimes interfering. 

Also, most times the pgn data is transferred correctly, but sometimes it shows up just empty, like if the game wasen't started. This may be the pgn data from the "unmoved" chessboard, and if so, this would indicate that there are in fact two games running... weird.  

Could this be a bug in game analysis?

**Comment by wormstein** [comment]
I'll have a look into high cpu usage and see what we can do about it. For sure it is related to raspi w as I never saw it happening on w2.

**Comment by EdNekebno** [comment]
In the latest push, I've changed the uci code so the second engine is only used for analysis when analysis mode is on. Pressing the down arrow turns the analysis off. That should help if we're right. I also changed it so that analysis mode is turned on if the os reports armv71 - which is on the Pi Zero 2W, I'm hoping that means it's turned off by default on the regular pi zero but I don't have one to hand to test.
